ui/public/index.html

```diff
@@ -8,6 +8,7 @@
       manifest.json provides metadata used when your web app is added to the
       homescreen on Android. See https://developers.google.com/web/fundamentals/engage-and-retain/web-app-manifest/
     -->
+    <script src="loadConfig.js"></script>
```

Comment:
nit: move above the manifest comment

Reply:
removed entirely and put into main react app.